terraso_backend/tests/graphql/test_shared_data.py

```diff
@@ -417,10 +414,28 @@ def test_data_entries_from_parent_query_by_resource_field(
     json_response = response.json()
 
     resources = json_response["data"][parent]["edges"][0]["node"]["sharedResources"]["edges"]
-    entries_result = [resource["node"]["source"]["name"] for resource in resources]
+    entries_result = [
+        {
+            "name": resource["node"]["source"]["name"],
+            "share_url": resource["node"]["shareUrl"],
+            "download_url": resource["node"]["downloadUrl"],
+            "share_access": resource["node"]["shareAccess"],
+        }
+        for resource in resources
+    ]
 
     for data_entry in data_entries:
-        assert data_entry.name in entries_result
+        uuid = data_entry.shared_resources.all()[0].share_uuid
+        download_url = f"{settings.API_ENDPOINT}/shared-data/download/{uuid}"
```

Comment:
issue: a helper like `resolve_download_url` every time you generate URLs of this form.